temporalio/converter.py

```diff
@@ -474,6 +475,12 @@ def encoding(self) -> str:
 
     def to_payload(self, value: Any) -> Optional[temporalio.api.common.v1.Payload]:
         """See base class."""
+        # Check for pydantic json callable thenpoe send warning
+        json_method = getattr(value, "json", None)
+        if json_method:
```

Comment:
I'm a little worried that someone will have a field/method named `json` on their class.

```suggestion
        if hasattr(value, "parse_obj"):
```